### ISSUE TITLE
cocopods fails due to working directory

### ISSUE THREAD

**Issue by paleozogt** [openbakery/gradle-xcodePlugin#356]
With 0.14.4/0.14.5, If I have a subproject that uses cocoapods, xcodePlugin falls down:
```:myproj:cocoapodsInstall
Run pod setup
Setting up CocoaPods master repo
  $ /usr/local/bin/git -C /Users/asimmons/.cocoapods/repos/master fetch origin --progress
  remote: Counting objects: 36, done.        
  remote: Compressing objects:   4% (1/21)           
  remote: Compressing objects:   9% (2/21)           
  remote: Compressing objects:  14% (3/21)           
  remote: Compressing objects:  19% (4/21)           
  remote: Compressing objects:  23% (5/21)           
  remote: Compressing objects:  28% (6/21)           
  remote: Compressing objects:  33% (7/21)           
  remote: Compressing objects:  38% (8/21)           
  remote: Compressing objects:  42% (9/21)           
  remote: Compressing objects:  47% (10/21)           
  remote: Compressing objects:  52% (11/21)           
  remote: Compressing objects:  57% (12/21)           
  remote: Compressing objects:  61% (13/21)           
  remote: Compressing objects:  66% (14/21)           
  remote: Compressing objects:  71% (15/21)           
  remote: Compressing objects:  76% (16/21)           
  remote: Compressing objects:  80% (17/21)           
  remote: Compressing objects:  85% (18/21)           
  remote: Compressing objects:  90% (19/21)           
  remote: Compressing objects:  95% (20/21)           
  remote: Compressing objects: 100% (21/21)           
  remote: Compressing objects: 100% (21/21), done.        
  remote: Total 36 (delta 22), reused 25 (delta 15), pack-reused 0        
  From https://github.com/CocoaPods/Specs
     ddc65a2..9803371  master     -> origin/master
  $ /usr/local/bin/git -C /Users/asimmons/.cocoapods/repos/master rev-parse --abbrev-ref HEAD
  master
  $ /usr/local/bin/git -C /Users/asimmons/.cocoapods/repos/master reset --hard origin/master
  HEAD is now at 9803371 [Add] Elemental 0.1.0
Setup completed
Run pod install
[!] No `Podfile' found in the project directory.
:myproj:cocoapodsInstall FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':myproj:cocoapodsInstall'.
> Command failed to run (exit code 1): '/usr/local/bin/pod install'
  Tail of output:
    remote: Compressing objects:  57% (12/21)           
    remote: Compressing objects:  61% (13/21)           
    remote: Compressing objects:  66% (14/21)           
    remote: Compressing objects:  71% (15/21)           
    remote: Compressing objects:  76% (16/21)           
    remote: Compressing objects:  80% (17/21)           
    remote: Compressing objects:  85% (18/21)           
    remote: Compressing objects:  90% (19/21)           
    remote: Compressing objects:  95% (20/21)           
    remote: Compressing objects: 100% (21/21)           
    remote: Compressing objects: 100% (21/21), done.        
    remote: Total 36 (delta 22), reused 25 (delta 15), pack-reused 0        
    From https://github.com/CocoaPods/Specs
       ddc65a2..9803371  master     -> origin/master
    $ /usr/local/bin/git -C /Users/asimmons/.cocoapods/repos/master rev-parse --abbrev-ref HEAD
    master
    $ /usr/local/bin/git -C /Users/asimmons/.cocoapods/repos/master reset --hard origin/master
    HEAD is now at 9803371 [Add] Elemental 0.1.0
  Setup completed
  [!] No `Podfile' found in the project directory.
```

Investigation shows that `/usr/local/bin/pod install` is being run with a working directory of ".", which is where `gradlew` was invoked from.

**Comment by paleozogt** [comment]
This seems to be similar to #312 which was also due to the working directory.

**Comment by phatblat** [comment]
@paleozogt did #312 resolve this issue?

**Comment by paleozogt** [comment]
I don't think so.  I'm seeing the problem in `0.14.5`, which presumably has #312 (it was in `0.14.4.develop.68`).

**Comment by phatblat** [comment]
I think this issue was resolved by #357
